react-backend/shared/controllers/feedback.js

```diff
@@ -1,24 +1,48 @@
-const cbAdmin = require('../models/cb-admin');
+const cbAdmin = require('../models/cb_admin');
 
-module.exports = (req, res, next) => {
-  const pgPool = req.app.get('client:psql');
+module.exports = {
+  get: (req, res, next) => {
+    const pgPool = req.app.get('client:psql');
+    try {
```

Comment:
You've wrapped this all in a try/catch. What exception are you expecting might be thrown?

Reply:
Just incase since/until are not date worthy

Reply:
Passing values that can't be parsed by the `Date` constructor won't cause it to  throw. Instead the postgres client will cause the promise to reject, which is covered by `catch`